### PR TITLE
Magnus/display mode font

### PR DESCRIPTION
**Pull request by mglyngberg**

### CODE REVIEW

**Comment by mglyngberg** [comment]
#178 was not merged into master btw

**Comment by bill-luu** [comment]
Can you edit your commit message so that it also reflects the thermometer widget removal?

**Comment by mglyngberg** [comment]
Can we just amend the commit message when we squash and merge?

**Comment by bill-luu** [comment]
Yeah that works too in this case, since it's just one commit

**Comment by bill-luu** [comment]
StateOfChargeIconWidget has `LCDMono` set in it's stylesheet.

File: src/ViewLayer/DisplayDashboard/DisplayDashboardUI/DisplayDashboardUI.cpp [hunk 1]
```diff
@@ -21,7 +21,7 @@ DisplayDashboardUI::DisplayDashboardUI()
     //Font loading
     QApplication::setFont(fontLoader_->loadFont(Font::BURLINGAME));
     ui_->actualSpeedLabel->setFont(fontLoader_->loadFont(Font::LCD, 45, true));
-    ui_->stateOfChargeCapacityWidget->setFont(fontLoader_->loadFont(Font::LCD, 20, false));
+    ui_->stateOfChargeCapacityWidget->setFont(fontLoader_->loadFont(Font::BURLINGAME, 20, false));
```
Comment (bill-luu) [inline src/ViewLayer/DisplayDashboard/DisplayDashboardUI/DisplayDashboardUI.cpp]: I meant that inside the `ui` file LCD was set in that stylesheet, you can undo this change :)
